examples/js/loaders/GLTFLoader.js

```diff
@@ -1547,59 +1523,57 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
-	function isArrayEqual( a, b ) {
+	function createGeometryKey( geometry ) {
```

Comment:
minor: could you name this parameter `primitiveDef` instead? I find it easier to distinguish whether we're working with the glTF primitive object or `THREE.Geometry` that way.